src/nav/apm.yml

```
@@ -1,16 +1,34 @@
 title: APM agents
 path: /docs/apm
 pages:
-  - title: Getting started
+  - title: Introduction to APM
+    path: /docs/apm/new-relic-apm/introduction-apm
+  - title: APM data security
+    path: /docs/apm/new-relic-apm/apm-agent-data-security
+  - title: Install language agents
+    pages:
+      - title: C install
```

```suggestion
      - title: C SDK install
```

Technically, it's the C SDK